wagtail/admin/views/reports.py

```diff
@@ -0,0 +1,30 @@
+from django.utils.translation import ugettext as _
+from django.views.generic.base import TemplateResponseMixin
+from django.views.generic.list import BaseListView
+
+from wagtail.core.models import UserPagePermissionsProxy
+
+
+class ReportView(TemplateResponseMixin, BaseListView):
+    header_icon = ''
+    page_kwarg = 'p'
+    template_name = None
+    title = ''
+    paginate_by = 10
+
+    def get_context_data(self, *args, object_list=None, **kwargs):
+        context = super().get_context_data(*args, object_list=object_list, **kwargs)
+        context['title'] = self.title
+        context['header_icon'] = self.header_icon
+        return context
+
+
+class LockedPagesView(ReportView):
+    template_name = 'wagtailadmin/reports/locked_pages.html'
+    title = _('Locked Pages')
```

Comment:
Should use `ugettext_lazy` here or this would be replaced with the translated version on module load. Probably easiest to update the import on line 1.

Reply:
Ah thanks, changed now.